go/enclave/crypto/transaction_blob_crypto.go

```diff
@@ -75,11 +77,23 @@ func (t TransactionBlobCryptoImpl) encrypt(transactions []*common.L2Tx) common.E
 		log.Panic("could not encrypt L2 transaction. Cause: %s", err)
 	}
 
-	return t.transactionCipher.Seal(nil, []byte(RollupCipherNonce), encodedTxs, nil)
+	nonce := make([]byte, NonceLength)
+	if _, err = io.ReadFull(rand.Reader, nonce); err != nil {
```

Comment:
worth checking here that this is the secure random. Or whether ego has an API for that.

cc @BedrockSquirrel

Reply:
Yes my understanding is this will use the SGX random number generator. EGo changes `crypt/rand` implementation to use it (https://github.com/edgelesssys/ego/issues/121#issuecomment-1053902171)

Reply:
Nice. that's cool.
It was worth a check

Reply:
Yeah I hadn't appreciated that tbh, good to know

Reply:
it's quite important. poor source of entropy can lead to hacks.